Source/WebCore/Modules/applepay/ApplePayRequestBase.cpp

```diff
@@ -103,7 +103,7 @@ ExceptionOr<ApplePaySessionPaymentRequest> convertAndValidate(Document& document
     result.setApplicationData(request.applicationData);
 
     if (version >= 3)
-        result.setSupportedCountries(WTFMove(request.supportedCountries));
+        result.setSupportedCountries(request.supportedCountries);
```

Comment:
Can we make the parameter a const ApplePayRequestBase& instead of a ApplePayRequestBase& to prevent other issues like this?

Reply:
Yeah good point! Fixed in https://github.com/WebKit/WebKit/commit/25ef05d703b8a89198b9eeea7f9dc648b8a5e71e